### PR TITLE
Create connect-two-servers-using-wireguard.md

### PR DESCRIPTION
**Pull request by fbrandstetter**
This article describes how a WireGuard connection between two servers can be established in order to share data, setup database replication and much more. Using NAT rules you can even let one server surf over the other!

### CODE REVIEW

File: docs/networking/vpn/connect-two-servers-using-wireguard.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+WireGuard is an extremely simple and fast VPN that utilizes state-of-the-art cryptography for highest security. The aim of the project is to be faster, simpler and more useful than IPSec. Currently, WireGuard is still in heavy development, altough it might already be regarded as the most secure and simplest VPN solution in the industry. WireGuard is useful for you if you want to exchange data securely between two servers.
+
+Before installing WireGuard, we assume that you have followed our [Getting Started Guide](/docs/getting-started/). If you're new to Linux server administration you may be interested in our [Introduction to Linux Concepts Guide](/docs/tools-reference/introduction-to-linux-concepts), [Beginner's Guide](/docs/beginners-guide/) and [Administration Basics Guide](/docs/using-linux/administration-basics). If you're concerned about securing on your Linode, you might be interested in our [Security Basics](/docs/security/basics) article as well.
+
+## Requirements
+This guide assumes that you're running the latest version of the Linux kernel, because WireGuard requires a kernel older than 4.1 in order to function correctly.
```
Comment (alexfornuto) [inline docs/networking/vpn/connect-two-servers-using-wireguard.md]: Older? As in an outdated kernel?

**Comment by gopalraha** [comment]
Have you accept NEW docs right now... If yes then I have some docs for Linode..

File: docs/networking/vpn/connect-two-servers-using-wireguard.md (hunk 1 above)
Comment (fbrandstetter) [inline docs/networking/vpn/connect-two-servers-using-wireguard.md]: Hello,
I am very sorry. By this, I meant the kernel should be newer than 4.1!

I'll try to fix that right now, sorry! :-(

Comment (fbrandstetter) [inline docs/networking/vpn/connect-two-servers-using-wireguard.md]: This should be fixed now.

**Comment by pbzona** [comment]
@fbrandstetter 

Thanks for your interest in contributing to our documentation. Unfortunately, we're unable to review or publish new guides right now due to the large backlog of pull requests. Please see our README for more information.

We'll make an announcement when the bounty program resumes, and I'd encourage you to consider sending us a new guide once that happens. Thanks again for your interest, but I'm going to be closing this PR for the reasons above.

**Pull request closed without merging**